scripts/create-darwin-volume.sh

```diff
@@ -127,42 +164,75 @@ main() {
     fi
 
     disk="$(root_disk_identifier)"
-    volume=$(find_nix_volume "$disk")
+    volume=$(find_nix_volume "$disk") # existing volname starting w/ "nix"?
     if [ -z "$volume" ]; then
-        echo "Creating a Nix Store volume..." >&2
+        volume="Nix Volume"    # otherwise use default
```

Comment:
Previously this was `Nix Store`. I'm partial to just calling it `Nix`, but I think `Nix Store` makes more sense than `Nix Volume` anyway; it's already a volume, so putting "volume" in the name is redundant.

Reply:
Good point. I was reacting to a comment about "Nix Store" not making sense because it holds more than just /nix/store, but didn't stop to consider the ATM machine I was setting loose.